### PR TITLE
Changes label and count focus for Items on Collection index (#1405).

### PR DESCRIPTION
**Pull request by bwatson78**
- app/presenters/hyrax/collection_presenter.rb: removes visibility constriction.
- config/locales/en.yml: changes header text.
- spec/presenters/collection_presenter_spec.rb: pulls in the spec from the Hyrax gem and focuses purely on changed method.
- spec/system/viewing_a_collection_spec.rb: adds expectations for the altered header.

### CODE REVIEW

**Comment by bwatson78** [comment]
Screen Shot 2020-08-14 at 2 15 26 PM

**Comment by devanshu-m** [comment]
This PR has work that's already in main. Can you please rebase?

File: spec/presenters/collection_presenter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
```
Comment (devanshu-m) [inline spec/presenters/collection_presenter_spec.rb:1]: Are we overwriting this spec? Can you mention what is being overwritten?

Comment (bwatson78) [reply spec/presenters/collection_presenter_spec.rb:1]: Just pushed a commit.

**Comment by bwatson78** [comment]
@devanshu-m Change made.

**Pull request merged**